tests/ui/option_as_ref_deref.stderr

```diff
@@ -88,5 +88,17 @@ error: called `.as_mut().map(DerefMut::deref_mut)` (or with one of deref aliases
 LL |     let _ = opt.clone().as_mut().map(|x| x.deref_mut()).map(|x| x.len());
    |             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ help: try using as_deref_mut instead: `opt.clone().as_deref_mut()`
 
-error: aborting due to 14 previous errors
+error: called `.as_ref().map(Deref::deref)` (or with one of deref aliases) on an Option value. This can be done more directly by calling `opt.as_deref()` instead
```

Comment:
@flip1995, Do I need to update `.as_ref().map(Deref::deref)` into `.as_ref().map(|x| &**x)` in this line?

Reply:
You can, if you want. It would be a nicer message. I'd merge this either way.

Reply:
Thanks, I'll try.